projects/table-builder/src/lib/classes/table-store.ts

```diff
@@ -97,6 +103,16 @@ export class TableStore extends ComponentStore<TableState> {
   });
 
 
+  setUserDefinedWidth = this.updater((state,colWidths:{key: string, widthInPixel:number, siblingWidthInPixel:number}[]) => {
```

Comment:
suggestion:
Let the directly call it twice for the column and the sibling instead of coupling them.
This opens a more flexible future where we may just want to expand the width of the table.